packages/kit/src/exports/vite/dev/index.js

```diff
@@ -176,20 +176,18 @@ export async function dev(vite, vite_config, svelte_config) {
 							const styles = {};
 
 							for (const dep of deps) {
-								const url = new URL(dep.url, 'http://localhost/');
+								if (/[?&](?:raw|url|inline)\b/.test(dep.url)) continue;
```

Comment:
this has a slight chance for false positives, move the check to after url parse and test query?

Reply:
Vite also does a loose check like this, so I don't think we need to refactor this. Though now it's checking by the search params instead which I'm also indifferent.